.travis.yml

```diff
@@ -16,4 +16,10 @@ jobs:
       name: "Unit Tests"
       script: npm run test:lint && npm run test:unit && npm run test:danger && npm run test:git-history
     - name: "Integration Tests"
-      script: ./scripts/run-integration-tests.travis.sh
\ No newline at end of file
+      script: ./scripts/run-integration-tests.travis.sh
+    - stage: release
```

Comment:
I don't think we should do this on every commit to master, we should do this on a new git tag or a similar process but I don't think we intend to release a new One App version on every new commit in master.